### DISCUSSION TITLE
Memos MS100F Smart Temperature and Humidity Sensor

### DISCUSSION THREAD

**Discussion by GalaxyEmperor** [krahabb/meross_lan#407] [General]
Hi Folks,
I'm new to this home assistant and home Automation stuff and really overwhelmed by this technology… so many technologies… so many standards… so many transmission technologies… it looks like the whole technology is reinvented for every new device 🤯.
I'm actually living in an Apple ecosystem and found my way to Home Assistant and I'm really at the start of building an automation system with a Raspberry PI. I guess the PI zero won't do the job but the 3B or 3B+ will be powerful enough for my little project. There is not really much to control, mainly a Smart Temperature and Humidity Sensor from Meross and a Dehumidifier. And there is my question… I have seen the Smart Temperature and Humidity Sensor MS100 is supported in the list of Supported hardware but not the MS100F, but found it in the 'meross_device_hub.py' at #821… there is the MS100FH mentioned. My research got me to this conclusion:
The MS100 family are the Humidity Sensor
the MS100 stands for the small square box
the MS100F is the round hokey puck like sensor with the solar cells
the MS100FH is the kit containing the round MS100F and the Smart Wi-Fi Hub MSH300HK
the H stands for Apple HomeKit

so I guess the above mentioned MS100FH in the 'meross_device_hub.py' refers rather to MS100F than the MS100FH and the smart WiFi Hub MSH300HK is not really needed.

Can someone confirm that?

**Comment by krahabb** [comment]
Hello @GalaxyEmperor,
I'm not totally aware of all the sudden naming logic behind Meross devices but in general, last letters of device type (like 'FH' for MS100FH) are just marketing variants (sometimes country specific) but, overall the devices are generally the same from a software perspective (take this with care as this is my experience so far with these naming variants)

`meross_lan` supports MS100 (and so should support MS100F) through the Meross Hub (MSH300 and the likes) which is anyway the hardware requirement needed to communicate to the MS100 sensor (this is for the official app too or for any communication infrastructure) since the Hub acts as a bridge between those small RF sensors/devices and your LAN.

By the look of the official Meross store I think your guessings are right but, as I've said, the Hub is neverless needed to communicate with these devices (all MS sensors for instance)
